config/301-awseventbridgetarget.yaml

```
@@ -0,0 +1,147 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: awseventbridgetargets.targets.triggermesh.io
+  labels:
+    knative.dev/crd-install: 'true'
+    triggermesh.io/crd-install: 'true'
+    duck.knative.dev/addressable: 'true'
+  annotations:
+    registry.triggermesh.io/acceptedEventTypes: |
+      [
+        { "type": "*" }
+      ]
+    registry.knative.dev/eventTypes: |
+      [
+        { "type": "io.triggermesh.targets.aws.eventbridge.result" }
+      ]
+spec:
+  group: targets.triggermesh.io
+  names:
+    kind: AWSEventBridgeTarget
+    plural: awseventbridgetargets
+    categories:
+    - all
+    - knative
+    - eventing
+    - targets
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    subresources:
+      status: {}
+    schema:
+      openAPIV3Schema:
+        type: object
+        description: TriggerMesh event target for AWS EventBridge Service.
+        properties:
+          spec:
+            description: Desired state of event target.
+            type: object
+            properties:
+              awsApiKey:
+                description: API Key to interact with the Amazon EventBridge API. For more information about AWS
+                  security credentials, please refer to the AWS General Reference at
+                  https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html
+                type: object
+                properties:
+                  secretKeyRef:
+                    description: A reference to a Kubernetes Secret object containing the access key ID.
+                    type: object
+                    properties:
+                      key:
+                        type: string
+                      name:
+                        type: string
+              awsApiSecret:
+                description: API Secret to interact with the Amazon EventBridge API. For more information about AWS
+                  security credentials, please refer to the AWS General Reference at
+                  https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html
+                type: object
+                properties:
+                  secretKeyRef:
+                    description: A reference to a Kubernetes Secret object containing the secret access key.
+                    type: object
+                    properties:
+                      key:
+                        type: string
+                      name:
+                        type: string
+              arn:
+                description: ARN of the EventBridge that will receive events. The expected format is documented at
```

Related to my previous comment.

```suggestion
                description: ARN of the Event Bus that will receive events. The expected format is documented at
```